### ISSUE TITLE
ModuleNotFoundError: No module named 'bounding_box'

### ISSUE THREAD

**Issue by h-stoyanov** [billstark/receipt-scanner#12]
The error occures on fresh install
```shell
$ python3 crnn_processor.py --image test.png 
Traceback (most recent call last):
  File "crnn_processor.py", line 5, in <module>
    from ReceiptGenerator.line_seg import cut_lines
  File "/home/icaka/Documents/Projects/test/receipt-scanner/ReceiptGenerator/line_seg.py", line 4, in <module>
    from bounding_box import BoundingBox
ModuleNotFoundError: No module named 'bounding_box'

```

**Comment by dutchwebs** [comment]
Try `pip install bounding_box`

**Comment by maminizer** [comment]
pip install bounding_box didn't work,

**Comment by Sandeep10021** [comment]
bounding_box is not a python module it's user's created

**Comment by beeceej** [comment]
I had to apply the following patch to get around these errors. @h-stoyanov  hopefully this is helpful.

```
diff --git a/ReceiptGenerator/letter_cutter.py b/ReceiptGenerator/letter_cutter.py
index 83099e6..bfd0dde 100644
--- a/ReceiptGenerator/letter_cutter.py
+++ b/ReceiptGenerator/letter_cutter.py
@@ -2,8 +2,8 @@ import numpy as np
 import cv2
 import os
 import sys
-from bounding_box import BoundingBox
-from utils import normalized_avg
+from ReceiptGenerator.bounding_box import BoundingBox
+from ReceiptGenerator.utils import normalized_avg

 def merge_bounding_boxes(bounding_boxes):
     # Algo to be improved for better performance
diff --git a/ReceiptGenerator/line_seg.py b/ReceiptGenerator/line_seg.py
index 7a87f19..5037cfa 100644
--- a/ReceiptGenerator/line_seg.py
+++ b/ReceiptGenerator/line_seg.py
@@ -1,8 +1,8 @@
 import cv2
 import numpy as np
 import os
-from bounding_box import BoundingBox
-from utils import normalized_avg
+from ReceiptGenerator.bounding_box import BoundingBox
+from ReceiptGenerator.utils import normalized_avg

 def eval_line_height(bounding_boxes):
     heights = [box.h for box in bounding_boxes]
```

The issue is that the modules `bounding_box` and `utils` are inside of the `ReceiptGenerator` module.

is this a patch that the maintainers would be willing to accept?

**Comment by h-stoyanov** [comment]
@beeceej I've originally deleted from my PC the downloaded repository. After I saw you fix I've downloaded it again and now I'm facing another issue.
```
pip install -r requirements.txt
ERROR: Could not find a version that satisfies the requirement tensorflow_gpu==1.1.0 (from versions: 2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.5.0, 2.5.1, 2.5.2, 2.6.0, 2.6.1, 2.6.2, 2.7.0rc0, 2.7.0rc1, 2.7.0, 2.8.0rc0, 2.8.0rc1)
ERROR: No matching distribution found for tensorflow_gpu==1.1.0
```
I tried tackle this issue by changing the version of the package but it requested changing the version of another packages and after successful install of all the requirements it didn't work at all.

**Comment by beeceej** [comment]
What version of python are you using? It looks like that this project will only support up to `python3.7.12` as written, since it relies on features that have been phased out, or changed in `tensorflow>2.0.0`, Which is not available in python versions after `3.7.12`. 

If you're able to switch to python 3.7.12, then patching the requirements with the code change [in this pull request](https://github.com/billstark/receipt-scanner/pull/22) should help

**Comment by h-stoyanov** [comment]
~~I've managed to install `python 3.7.12` via https://github.com/pyenv/pyenv-installer and created a virtual environment (venv) with this interpreter and the problems still occur~~
```shell
receipt-scanner$ ./venv/bin/python --version
Python 3.7.12
```
```shell
receipt-scanner$ ./venv/bin/pip install -r requirements.txt 
Collecting opencv-python
  Using cached opencv_python-4.5.5.62-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (60.4 MB)
Collecting augmentor
  Using cached Augmentor-0.2.9-py2.py3-none-any.whl (38 kB)
Collecting pillow
  Using cached Pillow-9.0.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.3 MB)
ERROR: Could not find a version that satisfies the requirement tensorflow_gpu==1.1.0 (from versions: 1.13.1, 1.13.2, 1.14.0, 1.15.0, 1.15.2, 1.15.3, 1.15.4, 1.15.5, 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.4.0, 2.4.1, 2.4.2, 2.4.3, 2.4.4, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.6.0, 2.6.1, 2.6.2, 2.6.3, 2.7.0rc0, 2.7.0rc1, 2.7.0, 2.7.1, 2.8.0rc0, 2.8.0rc1, 2.8.0)
ERROR: No matching distribution found for tensorflow_gpu==1.1.0
WARNING: You are using pip version 21.1.2; however, version 22.0.3 is available.
You should consider upgrading via the '/home/icaka/Documents/Projects/receipt-scanner/venv/bin/python -m pip install --upgrade pip' command.
```
Edit: I used the code from your fork with the specified branch and now it works as expected! Thank you! I will start testing with Cyrillic  characters a soon as possible :)
